src/components/TransactionDialogs/SendDialog/index.tsx

```diff
@@ -0,0 +1,138 @@
+import React from 'react'
+import { useForm } from 'react-hook-form'
+import { ReactComponent as RemoveIcon } from '../../../assets/images/icons/icon_clear.svg'
+import useIconProps from '../../../misc/useIconProps'
+import Dialog from '../../Element/dialog'
+import Button from '../../Element/button'
+import useStateHistory from '../../../misc/useStateHistory'
+
+interface Props {
+  open: boolean
+  onClose: () => void
+  account: Account
+}
+
+export enum Stage {
+  SendStage = 'Send',
+  ConfirmTxStage = 'Send',
```

Comment:
duplicate enum value

Reply:
I saw from the UI that the `Send` title last for 2 stages, for now I'll delete the `ConfirmTxStage`

Reply:
this enum is not title. this is simply an id for it to distinguish different stage. it can actually be 0, 1, 2, etc, but for readability we use a string value to describe the stage